specification/hugr.md

```diff
@@ -1239,17 +1246,16 @@ The `Replace` method takes as input:
 
 The new hugr is then derived as follows:
 
-1.  Make a copy in $\Gamma$ of all the nodes in $G$, and all edges between them.
-2.  For each $\sigma\_\mathrm{inp} \in \mu\_\textrm{inp}$, insert a new edge going into the new
+1.  Make a copy in $\Gamma$ of all the nodes in $H$ *except the root*, and all edges where both source and target are copied.
```

Comment:
Isn't "where both source and target are copied" redundant? The only node that isn't copied is the root...

Reply:
Yes, good point...:-)

Reply:
Clarified